docs/content/preview/migrate/migrate-steps.md

```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
 
 The options passed to the command are similar to the [`yb-voyager export schema`](#export-schema) command. To export only a subset of the tables, pass a comma-separated list of table names in the `--table-list` argument.
 
+#### Accelerate data export for MySQL or Oracle
+
+If you want to speed up data export for MySQL/Oracle, you can set the environment variable `EXPERIMENTAL_FAST_EXPORT=1` when you run `export data` using yb-voyager.
+
+Consider the following caveats before using the feature:
+
+- You need to perform additional steps when you [prepare the source database](#prepare-the-source-database).
+- Some data types are unsupported. For a detailed list, refer to [datatype mappings](../reference/datatype-mapping-mysql/).
+- [--parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument will have no effect.
+- SSL is unsupported.
+- In MySQL RDS, writes are not allowed during the data export process.
+
+{{< note title="Sequence migration considerations" >}}
```

Comment:
what's the link? if EXPERIMENTAL_FAST_EXPORT is using sequence migration, that should be worked into/explained in the opening paragraph of this section, rather than being brought up in an otherwise unrelated-looking note at the end of the section

Reply:
Agreed, it is unrelated. @aishwarya24 can you pls move this to a separate section similar to `Accelerate data export for MySQL or Oracle`

Reply:
@aishwarya24 I believe this is still a note. Can you please convert it to a separate section as mentioned? Thanks